### DISCUSSION TITLE
Why is sandbox making requests to so many 3rd parties?

### DISCUSSION THREAD

**Discussion by anddoutoi** [apollographql/apollo-server#7339] [Q&A]
Using `ApolloServerPluginLandingPageLocalDefault` plugin, loading localhost:4000, clicking on the button "Query your server"and loading https://studio.apollographql.com/sandbox/explorer.

Why are there so many many requests? Many requests seems to be totally unnecessary.

- https://bat.bing.com
- https://googleads.g.doubleclick.net
- https://www.facebook.com
- https://analytics.twitter.com
- https://t.co/1/i/adsct
- https://px.ads.linkedin.com
- https://events.launchdarkly.com
- https://js.stripe.com
- https://m.stripe.com
- https://api.segment.io
- https://app.satismeter.com

**Comment by trevor-scheer** [comment]
Some of these requests I don't see at all on my end (bing, twitter, linkedin) - do you still see them? Can you confirm it's none of your browser extensions?

A couple of these are actually important for the functionality of Sandbox (Stripe for onboarding, LaunchDarkly for feature flags)

I did raise this to the team and there should be no more analytics/tracking when you're logged in to Studio (this was a bug), but we do use analytics for non-logged in users.
